bobtimus/src/bin/fake_bobtimus.rs

```diff
@@ -39,9 +47,18 @@ async fn main() -> Result<()> {
         .allow_methods(vec!["GET"])
         .allow_header("content-type");
 
-    let latest_rate = warp::path("rate")
-        .and(warp::get())
-        .map(|| warp::sse::reply(warp::sse::keep_alive().stream(fixed_rate_stream())));
+    let latest_rate = warp::path("rate").and(warp::get()).map(|| {
```

Comment:
I think we should add a path segment here that says this rate is for `LBTC-LUSDT`.